internal/handler/msg_hello.go

```diff
@@ -48,7 +48,7 @@ func (h *Handler) MsgHello(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 	var reply wire.OpMsg
 	resp := must.NotFail(types.NewDocument(
 		"isWritablePrimary", true,
-		"maxBsonObjectSize", int32(types.MaxDocumentLen),
+		"maxBsonObjectSize", int32(h.MaxBsonObjectSizeBytes),
 		"maxMessageSizeBytes", int32(wire.MaxMsgLen),
```

Comment:
Not in scope of this issue, there is `wire.MaxMsgLen` which is 48MB. So currently entire message size is limited by this https://github.com/chilagrow/FerretDB/blob/4db0c5e5ef79edd7fbe654b173a0e804011a7465/internal/wire/msg_header.go#L99

I'm noting that if this check on wire level is something we might want to change in the future?